tests/integration/components/modals/event-delete-modal-test.js

```diff
@@ -0,0 +1,13 @@
+import { test } from 'ember-qunit';
+import moduleForComponent from 'open-event-frontend/tests/helpers/component-helper';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent('modals/event-delete-modal', 'Integration | Component | modals/event delete modal');
+
+test('it renders', function(assert) {
+  this.set('isOpen', false);
+  this.set('eventName', 'sample');
+  this.set('deleteEvent', 'deleteEvent');
```

Comment:
@niranjan94 Since the action is not doing anything right now, will the test action be empty?